ShinyExpenseTracker/src/ca/ualberta/cs/shinyexpensetracker/activities/AddDestinationActivity.java

```diff
@@ -50,6 +53,8 @@
 	
 	private Destination destination;
 	private int destinationIndex;
+	
+	private Coordinate coord = new Coordinate();
```

Comment:
Any reason why this isn't just `null` at the start? Would make the check at line 133 make a little more sense, as it could just check if it's null or not.